bin.js

```diff
@@ -26,9 +26,9 @@ var argv = minimist(process.argv.splice(2), {
 })
 
 function usage () {
-  console.log('Usage: pes-client SCHEMA [--port PORT] [--host HOST]\n' +
-              '                  [--target-host HOST] [--target-port PORT]\n' +
-              '                  [--db PATH] [--no-repl] [--from-scratch]')
+  console.log('Usage: hypem <SCHEMA> [--schema SCHEMA] [--port PORT] [--host HOST]\n' +
```

Comment:
@mcollina How do I say, using that notation, that I need __either__ SCHEMA or [--schema], I need one or the other, I misread and thought `<SCHEMA>` meant this.